fluid-music/src/tracksToReaperProject.js

```diff
@@ -0,0 +1,191 @@
+const rppp = require('rppp')
+const tab = require('./tab')
+
+// Requires a bpm
+
+
+/**
+ * Create a `ReaperProject` from a `TracksObject`
+ *
+ * ```javascript
+ * const session = fluid.score.parse(myScore, myConfig);
+ * const message = fluid.score.tracksToFluidMessage(session.tracks);
+ * ```
+ *
+ * @param {TracksObject} tracksObject A tracks object generated by score.parse
+ * @returns {ReaperProject}
+ */
+function tracksToReaperProject(tracksObject) {
+
+  let bpm = 120;
+  if(tracksObject.bpm) bpm = tracksObject.bpm;
+
+  let i = 0;
+  const reaperProject = new rppp.objects.ReaperProject();
+  reaperProject.getOrCreateStructByToken('TEMPO').params = [bpm, 4, 4];
+
+  // // example tracks object
+  // const tracks = {
+  //   bass: { clips: [ clip1, clip2... ] },
+  //   kick: { clips: [ clip1, clip2... ] },
+  // };
+  for (const [trackName, track] of Object.entries(tracksObject)) {
+    if (tab.reservedKeys.hasOwnProperty(trackName)) {
+      continue;
+    }
+
+    if (!track.clips || !track.clips.length) {
+      if (!tracksObject.plugins.length) {
+        console.log(`tracksToFluidMessage: skipping ${trackName}, because it has no .clips and no .plugins`);
+        continue;
+      }
+    }
+
+    // Create a sub-message for each track
+    let newTrack = new rppp.objects.ReaperTrack();
+    newTrack.getOrCreateStructByToken('NAME').params[0] = trackName;
+
+    track.clips.forEach((clip, clipIndex) => {
+
+      // Create one EventContext object for each clip.
+      const context = {
+        bpm,
```

Comment:
`bpm` should be added as an optional property to the ClipEventContext JSDoc in `./src/jsdoc-types.js`. Include a comment explaining that it is used in the reaper serializer.